sdk/monitor/monitor-query/test/public/logsQueryClient.spec.ts

```diff
@@ -527,7 +527,7 @@ describe("LogsQueryClient live tests - server timeout", function () {
   });
   // disabling http retries otherwise we'll waste retries to realize that the
   // query has timed out on purpose.
-  it("serverTimeoutInSeconds", async function () {
+  it.skip("serverTimeoutInSeconds", async function () {
```

Comment:
Can you please create a separate GitHub issue to investigate and find a long term solution for this issue?

Reply:
I will keep the same issue open. I expect a long term solution early next week.